query/query.go

```diff
@@ -230,25 +230,27 @@ type Function struct {
 // SubGraph is the way to represent data. It contains both the request parameters and the response.
 // Once generated, this can then be encoded to other client convenient formats, like GraphQL / JSON.
 // SubGraphs are recursively nested. Each SubGraph contain the following:
-// * SrcUIDS: A list of UIDs that were sent to this query. If this subgraph is a child graph, then the
```

Comment:
Are we changing anything significant in the comments?

Reply:
No, we are not. But these are go fmt changes which can be safely accepted.